lib/Install.js

```diff
@@ -117,6 +122,122 @@ function install(
   return 'SuccessfullyInstalled';
 }
 
+async function downloadFileNative(url, filePath) {
+  const usedCommand = `require("https").get(${JSON.stringify(url)})`;
+  const errorPrefix = `${usedCommand}\nThe above call errored: `;
+
+  return new Promise((resolve, reject) => {
+    const file = fs.createWriteStream(filePath);
+    let fileInfo = null;
+
+    const request = https.get(url, response => {
+      if (response.statusCode !== 200) {
+        fs.unlink(filePath, () => {
+          reject(new Error(`Failed to get '${url}' (${response.statusCode})`));
+        });
+        return;
+      }
+
+      fileInfo = {
+        mime: response.headers['content-type'],
+        size: parseInt(response.headers['content-length'], 10),
+      };
+
+      response.pipe(file);
+    });
+
+    // The destination stream is ended by the time it's called
+    file.on('finish', () => resolve(fileInfo));
+
+    request.on('error', err => {
+      fs.unlink(filePath, () => reject(err));
+    });
+
+    file.on('error', err => {
+      fs.unlink(filePath, () => reject(err));
+    });
+
+    request.end();
+  });
+}
+
+async function installUnstableTestMaster(projectRootDir) {
+  console.log(chalk.yellow('Using the master version of elm-explorations/test'));
+  console.log(chalk.yellow('Note you might need to rm -rf ~/.elm and/or ./elm-stuff afterwards.'));
+
+  const pkg = 'elm-explorations/test';
+  const version = '1.2.2';
+  const pkgWithDash = pkg.replace('/','-');
+
+  const tempPath = path.join(
+    projectRootDir,
+    'elm-stuff',
+    'generated-code',
+    'elm-community',
+    'elm-test'
+  );
```

Comment:
If you take a `Project` as a parameter instead of just the project root dir, you can use `project.generatedCodeDir` instead of repeating this.